rules/Php71/Rector/FuncCall/RemoveExtraParametersRector.php

```diff
@@ -146,8 +146,8 @@ private function resolveMaximumAllowedParameterCount(
         MethodReflection | FunctionReflection $functionLikeReflection
     ): int {
         $parameterCounts = [0];
-        foreach ($functionLikeReflection->getVariants() as $parametersAcceptor) {
-            $parameterCounts[] = count($parametersAcceptor->getParameters());
+        foreach ($functionLikeReflection->getVariants() as $parameterAcceptor) {
```

Comment:
@TomasVotruba @staabm for this case, it is object named `ParametersAcceptor` which keep to be named as `$parametersAcceptor` instead of `$parameterAcceptor` seems ok.

I think for object type, the naming should not singularize, but just make it snake case? @staabm could you take a look as well?

Reply:
This should be kept as `$parametersAcceptor` to respect `ParametersAcceptor` type.